src/loader/Pix3DLoader.py

```diff
@@ -0,0 +1,107 @@
+import glob
+import json
+import os
+import random
+
+from src.loader.LoaderInterface import LoaderInterface
+from src.utility.Utility import Utility
+from src.utility.LabelIdMapping import LabelIdMapping
+
+
+class Pix3DLoader(LoaderInterface):
+    """
+    This loads an object from Pix3D based on the given category of objects to use.
+
+    From these objects one is randomly sampled and loaded.
+
+    As for all loaders it is possible to add custom properties to the loaded object, for that use add_properties.
+
+    Finally it sets all objects to have a category_id corresponding to the void class, 
+    so it wouldn't trigger an exception in the SegMapRenderer.
+
+    Note: if this module is used with another loader that loads objects with semantic mapping, make sure the other module is loaded first in the config file.
+
+    **Configuration**:
+
+    .. csv-table::
+       :header: "Parameter", "Description"
+
+       "data_path", "The path to the Pix3D folder. Type: string."
+       "category", "The category to use for example: 'bed', check the data_path/img, data_path/mask or data_path/model folder for more categories. Type: string."
+    """
+
+    def __init__(self, config):
+        LoaderInterface.__init__(self, config)
+
+        self._data_path = Utility.resolve_path(self.config.get_string("data_path"))
```

Comment:
Can we use, here as the default the resource folder, where the data is put if the download script was used.